### ISSUE TITLE
T2V working But I2V doesn't work !

### ISSUE THREAD

**Issue by aleou** [raindrop313/ComfyUI-WanVideoStartEndFrames#13]
Failed running call_function <built-in function linear>(*(FakeTensor(..., device='cuda:0', size=(769, 4863), dtype=torch.float16), FakeTensor(..., device='cuda:0', size=(5120, 5120), dtype=torch.float16), FakeTensor(..., device='cuda:0', size=(5120,), dtype=torch.float16)), **{}):
a and b must have same reduction dim, but got [769, 4863] X [5120, 5120].

from user code:
File "/comfyui/custom_nodes/ComfyUI-GGUF/ops.py", line 215, in torch_dynamo_resume_in_forward_ggml_cast_weights_at_214
return torch.nn.functional.linear(input, weight, bias)

Set TORCH_LOGS="+dynamo" and TORCHDYNAMO_VERBOSE=1 for more information

You can suppress this exception and fall back to eager by setting:
import torch._dynamo
torch._dynamo.config.suppress_errors = True

---

Here is the worflow :

[Wan Image to Video Fast Workflow v1 - I2V doesn't work.json](https://github.com/user-attachments/files/19448119/Wan.Image.to.Video.Fast.Workflow.v1.-.I2V.doesn.t.work.json)

**Comment by aleou** [comment]
after some testing the SLG node is the origine of this error !
idk why.

**Comment by raindrop313** [comment]
The current frame workflow likely does not support GGUF. You can first try using the workflow provided by this project to see if it works, and then expand upon it accordingly.

> after some testing the SLG node is the origine of this error ! idk why.
